config/nav.yml

```diff
@@ -137,6 +137,13 @@ nav:
           - Configuring gradual rollout of traffic to Revisions: serving/configuration/rolling-out-latest-revision-configmap.md
           - Feature and extension flags: serving/configuration/feature-flags.md
           - Configure the Defaults ConfigMap: serving/configuration/config-defaults.md
+      # Serving - Application Security
```

Comment:
https://deploy-preview-5256--knative.netlify.app/docs/serving/app-security/security-guard-using/ the "Using Security-Guard" section should be merged with the "About" section - it doesn't make sense to have more conceptual information after all the procedures